test/buffer_pool_test.cc

```diff
@@ -85,7 +85,7 @@ void TestGetBandwidths(hermes::SharedMemoryContext *context) {
   }
 }
 
-hapi::Status ForceBlobToSwap(Hermes *hermes, hermes::u64 id, hapi::Blob &blob,
+hermes::Status ForceBlobToSwap(Hermes *hermes, hermes::u64 id, hapi::Blob &blob,
```

Comment:
If you search the code for `hermes::Status` you'll find 7 hits. Could you please change those to `hapi::Status`? The reason it compiles is because `data_placement_engine.h` and `data_placement_engine.cc` both have `using api::Status;` statements in the `hermes` namespace.

Reply:
Done!